libs/payments/stripe/src/lib/stripe.client.spec.ts

```diff
@@ -8,13 +8,32 @@ import { InvoiceFactory } from './factories/invoice.factory';
 import { SubscriptionListFactory } from './factories/subscription.factory';
 import { StripeClient } from './stripe.client';
 
+jest.mock('stripe', () => ({
+  Stripe: function () {
+    return {
+      customers: {
+        retrieve: jest.fn(),
+      },
+      invoices: {
+        finalizeInvoice: jest.fn(),
+      },
+      subscriptions: {
+        list: jest.fn(),
+      },
+    };
+  },
+}));
+
```

Comment:
Can we let Jest automock this rather than specifying properties?

AKA just `jest.mock('stripe');`